src/test/ui/const-generics/conservative_is_privately_uninhabited_uses_correct_param_env-1.rs

```diff
@@ -1,28 +0,0 @@
-// run-pass
```

Comment:
Is it okay to delete these tests? I just don't know what they are testing for anymore since they are named after the function being removed.

Reply:
I think we should keep them. They are going through the regular code path now instead of the conservative one, and that one shouldn't ICE either.